lib/settings_base.py

```diff
@@ -229,6 +229,11 @@ def lazy_langs(languages):
 # Tarballs in DUMPED_USERS_PATH deleted 30 days after they have been written.
 DUMPED_USERS_DAYS_DELETE = 3600 * 24 * 30
 
+# path that isn't just one / deep, and doesn't require any locale or app.
+SUPPORTED_NONAPPS_NONLOCALES_DEEP = (
```

Comment:
Also a pet peeve of mine is using tuples for non-tuply things instead of a list. I know other people are totally cool with this but I feel like this should be a list since you would never index into it looking to take a specific entry like a tuple of `(prefix, supported_methods)`.

Obviously not a big deal.